charts/gardener/controlplane/charts/application/templates/apiservice-v1alpha1-security-gardener-cloud.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1alpha1.authentication.gardener.cloud
+  name: v1alpha1.security.gardener.cloud
```

Comment:
Add another test case in https://github.com/gardener/gardener/blob/10d75787a132bf39c408b74372f5d8c045fa8f4b/test/integration/envtest/environment_test.go#L20-L43 for the new API group?